charts/tidb-operator/templates/advanced-statefulset-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
```

Comment:
I'll add an option in helm chart to deploy advanced statefulset when the feature is enabled.

Reply:
done